include/RandLAPACK/comps/determiter.hh

```diff
@@ -10,21 +10,21 @@ namespace RandLAPACK::comps::determiter {
 
 template <typename T>
 void pcg(
-	int64_t m,
-	int64_t n,
-	T* const A,
-	int64_t lda, 
-	T* const b, // length m
-	T* const c, // length n
+    int64_t m,
+    int64_t n,
+    T* const A,
```

Comment:
`T * const A` means the pointer value A is not modifiable. `const T *A` or `T const *A` means the data pointed to by A is not modifiable. One of the latter two options is what is needed here. As a matter of of the project's code style I suggest we adopt the first, `const T *A`.

Reply:
Agreed. This is an issue i inherited. Fixed now.